src/test/shamap/SHAMapSync_test.cpp

```diff
@@ -194,7 +194,7 @@ class sync_test : public beast::unit_test::suite
 
         BEAST_EXPECT(source.deepCompare (destination));
 
-        std::cerr << "Checking destination invariants" << std::endl;
+        log << "Checking destination invariants..." << std::endl;
```

Comment:
I think the log message is probably not needed - we can instead use the BEAST_EXPECTS macro to pinpoint a failure in checkInvariants - will investigate more.